notifier/src/main/java/com/epam/dojo/notifier/configuration/RestApiConfiguration.java

```diff
@@ -5,32 +5,15 @@
 
 @ConfigurationProperties
 @Component
-public class Configuration {
+public class RestApiConfiguration {
+
     private String leaderboardApi;
     private String userDetailsApi;
-    private int threadPoolSchedulePeriodSeconds;
-    private int threadPoolSize;
-
```

Comment:
Let's keep threadPool properties. They are wired with @Value in NotificationManagingService, but still part of the configuration.